docs/main/user-guides/kubectl-openebs.md

```diff
@@ -0,0 +1,202 @@
+---
+id: kubectl-openebs
+title: Kubectl OpenEBS Plugin
+keywords:
+ - Kubectl
+ - Plugin
+ - Kubectl Plugin
+ - Kubectl OpenEBS Plugin
+description: A comprehensive guide to the kubectl-openebs plugin, enabling unified management of OpenEBS storages with enhanced features and improved usability.
+---
+# Kubectl OpenEBS Plugin
+
+## Overview
+
+The Kubectl OpenEBS Plugin is an OSS plugin that allows you to interact with all OpenEBS storages through a unified interface. This unified plugin is especially beneficial if you have installed a cluster using the OpenEBS umbrella chart. The supported storages include:
+
+- Local PV HostPath
+- Local PV LVM
+- Local PV ZFS
+- Replicated PV Mayastor
+
+This plugin integrates the current `kubectl-mayastor` functionality for Replicated PV Mayastor specific commands while introducing newly developed functionality for other storages, inspired by the existing `kubectl-openebs` plugin. The previous `kubectl-openebs` supported only Local PV storages whereas now it extends its capabilities to include Replicated PV Mayastor commands. Also, the plugin supports multiple output formats, such as YAML and JSON.
+
+By using this plugin, you can gain a simplified and consistent interface to manage multiple storages, with improved output formats, flexible namespace configurations, and robust command options tailored to meet diverse storage needs.
+
+## Features
+
+### Unified Plugin for All Storages
+
+The plugin consolidates functionality for all OpenEBS storages under a single interface. You can manage Local PV Hostpath, Local PV LVM, Local PV ZFS, and Replicated PV Mayastor resources without switching between multiple tools or plugins.
+
+### Namespace Assumption
```

Comment:
Can you refer this section from README.md? I have made some changes there.

https://github.com/openebs/openebs/blob/develop/plugin/README.md#namespace-handling